src/plugins/integrations/PostgreIntegration.ts

```diff
@@ -41,66 +43,69 @@ class PostgreIntegration implements Integration {
   wrap(lib: any, config: any) {
     function wrapper(query: any, args: any) {
       return function queryWrapper() {
-
-        const tracer = ThundraTracer.getInstance();
-        const parentSpan = tracer.getActiveSpan();
-
-        const params = this.connectionParameters;
-        const span = tracer._startSpan(params.database, {
-          childOf: parentSpan,
-          domainName: DomainNames.DB,
-          className: ClassNames.RDB,
-          disableActiveStart: true,
-        });
-
-        if (params) {
-          span.addTags({
-            [SpanTags.SPAN_TYPE]: SpanTypes.RDB,
-            [DBTags.DB_INSTANCE]: params.database,
-            [DBTags.DB_USER]: params.user,
-            [DBTags.DB_HOST]: params.host,
-            [DBTags.DB_PORT]: params.port,
-            [DBTags.DB_TYPE]: DBTypes.PG,
-            [SpanTags.TOPOLOGY_VERTEX]: true,
-            [SpanTags.TRIGGER_DOMAIN_NAME]: LAMBDA_APPLICATION_DOMAIN_NAME,
-            [SpanTags.TRIGGER_CLASS_NAME]: LAMBDA_APPLICATION_CLASS_NAME,
-            [SpanTags.TRIGGER_OPERATION_NAMES]: [tracer.functionName],
+        try {
+          const tracer = ThundraTracer.getInstance();
+          const parentSpan = tracer.getActiveSpan();
+
+          const params = this.connectionParameters;
+          const span = tracer._startSpan(params.database, {
+            childOf: parentSpan,
+            domainName: DomainNames.DB,
+            className: ClassNames.RDB,
```

Comment:
I think, anymore we can set class name to the `DBTypes. PG` here instead of `RDB` constant.